entity-service/.env.example

```
@@ -18,3 +18,14 @@ SERVICENOW_INTEGRATION_SERVICE_TOKEN_URL=<SERVICENOW_INTEGRATION_SERVICE_TOKEN_U
 SERVICENOW_INTEGRATION_SERVICE_CLIENT_ID=<SERVICENOW_INTEGRATION_SERVICE_CLIENT_ID>
 SERVICENOW_INTEGRATION_SERVICE_CLIENT_SECRET=<SERVICENOW_INTEGRATION_SERVICE_CLIENT_SECRET>
 SERVICENOW_INTEGRATION_SERVICE_SCOPES=<SERVICENOW_INTEGRATION_SERVICE_SCOPES>
+
+# Team registry — the curated account-based-team vocabulary, as
+# "teamKey|Display Name|FAMILY" rows separated by commas. The family field is
+# optional and is CRE or SRE. Display Name must match the team's group name in
+# the backing data source exactly. No default: unset means no teams are
+# configured, and the values below are placeholders, not real team names.
+CSM_TEAM_REGISTRY=alpha|Alpha Team|CRE,beta|Beta Team|SRE,gamma|Gamma Team
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)